### ISSUE TITLE
Document didn't have any nodes

### ISSUE THREAD

**Issue by beeing** [RazrFalcon/svgcleaner#100]
Hi, I've tried the older svgcleaner 0.6.2 on a pdf which works but on the latest version have the error above.

**Comment by RazrFalcon** [comment]
Can you provide an example file?

**Comment by beeing** [comment]
[Archive.zip](https://github.com/RazrFalcon/svgcleaner/files/1505306/Archive.zip)
I've included both the cleaned (from 0.6.2) and original svg.

**Comment by RazrFalcon** [comment]
Explicit namespaces will be supported in the next release.

**Comment by beeing** [comment]
Meaning that the nodes error is due to namespaces?

Anyway, thanks for the help. Looking forward to the next release.

**Comment by RazrFalcon** [comment]
Yes. Current version of the xml parser doesn't understand that `svg:svg` is the same as `svg`. Still, it's a very rare format.
